app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WindowWidget.java

```diff
@@ -660,17 +660,10 @@ public void disableVRVideoMode() {
     }
 
     public void setWindowPlacement(@NonNull Windows.WindowPlacement aPlacement) {
-        if (mActive) {
-            TelemetryWrapper.activePlacementEvent(mWindowPlacement.getValue(), false);
-        }
-
```

Comment:
@keianhzo This looks like a bug, right? Why do we set `activePlacementEvent` false when aPlacement is true?

Reply:
It's being called also in `setActiveWindow `, the problem is that when a window placement changes only `setWindowPlacement` is called so we need to also update the active ties there.